patches/build-android-gyp-util-resource_utils.py.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/build/android/gyp/util/resource_utils.py b/build/android/gyp/util/resource_utils.py
+index 8105801deb1009614d0d18470b14d531557933b6..7ea52e603be06458fbf74aadd67ab722ea2c0587 100644
+--- a/build/android/gyp/util/resource_utils.py
++++ b/build/android/gyp/util/resource_utils.py
+@@ -235,6 +235,7 @@ class ResourceInfoFile(object):
+       return
+     source_path = os.path.normpath(source_path)
+     new_value = self._entries.setdefault(archive_path, source_path)
++    return
```

Comment:
this intentionally skips the duplicate key check because we are overwriting some of the files